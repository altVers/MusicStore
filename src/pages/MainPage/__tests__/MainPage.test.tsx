import { render } from "@testing-library/react";
import { MainPage } from "../MainPage";

describe('Текст по отрисовке MainPage', () => {
    test('Текст по отрисовке MainPage', () => {
        expect(render(<MainPage />)).toMatchSnapshot()
    })
})