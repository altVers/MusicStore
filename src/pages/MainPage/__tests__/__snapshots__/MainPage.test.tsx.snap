// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Текст по отрисовке MainPage Текст по отрисовке MainPage 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mainPage"
      >
        <h2>
          MainPage
        </h2>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nam cumque eligendi nobis reprehenderit delectus veritatis fuga ipsa vitae aliquid. Totam dicta quis iusto vero nisi et ex cupiditate corporis architecto?
        </p>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem culpa assumenda laudantium libero sapiente vitae ut maiores quod ad, ducimus eius dicta et, odio, beatae maxime voluptatem officiis unde molestiae.
        </p>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi accusamus, corporis suscipit non quaerat aspernatur maxime voluptatum totam voluptates omnis saepe similique aut earum dolor necessitatibus quia quam, porro architecto!
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mainPage"
    >
      <h2>
        MainPage
      </h2>
      <p>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nam cumque eligendi nobis reprehenderit delectus veritatis fuga ipsa vitae aliquid. Totam dicta quis iusto vero nisi et ex cupiditate corporis architecto?
      </p>
      <p>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem culpa assumenda laudantium libero sapiente vitae ut maiores quod ad, ducimus eius dicta et, odio, beatae maxime voluptatem officiis unde molestiae.
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi accusamus, corporis suscipit non quaerat aspernatur maxime voluptatum totam voluptates omnis saepe similique aut earum dolor necessitatibus quia quam, porro architecto!
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
